azure_arc_servers_jumpstart/automanage/artifacts/arcAutomanage.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Environment Variables
+export automanageAccountName='<Name of your Azure Automanage Account>'
+export location='<Azure Region where your Azure Arc enabled Server is registered>'
+export resourceGroup='<Azure Resource Group where your Azure Arc enabled server is registered>'
+export machinename='<Name of your Azure Arc enabled Server>'
+export profile='<environment of your Azure Arc enabled Server Production or Dev/Test>'
+
+# Replace environment variables in parameters file
+
+sed -i "s/<name of your Automanage Account identity>/$automanageAccountName/" automanageAccount.parameters.json
```

Comment:
this should be an environment variable edited in the main script block above instead of user having to edit the sed line directly. It could be confusing for a user that is not that familiar with shell scripts.